ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -1201,7 +1231,7 @@ ORK_CLASS_AVAILABLE
  
  When the value of this property is 0, there is no maximum.
  */
-@property (readonly) NSInteger maximumLength;
+@property (readwrite) NSInteger maximumLength;
```

Comment:
Since we are switching to *readwrite*, we do not need to specify it. The default is *readwrite*.